### PR TITLE
add polar stereo projection

### LINKED ISSUE #55: Add polar stereo projection
## Description

This issue is to accomplish two things:

-  Pass the projection and domain argument down the lower code levels

- Add  polar stereo-graphic projections

### PR DESCRIPTION
**Pull request by mjhossen**
## Description

This PR adds polar stereo projection to plot polar regions, such as north and south pole. 

## Issue(s) addressed
Link the issues to be closed with this PR
- fixes #55

### CODE REVIEW

File: src/eva/plot_tools/figure.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def _map_scatter(self, plotobj, ax):
 
             cs = ax.scatter(plotobj.longitude, plotobj.latitude,
                             s=plotobj.markersize, **inputs,
-                            transform=self.projection.projection)
+                            transform=ccrs.PlateCarree())
```
Comment (danholdaway) [inline src/eva/plot_tools/figure.py]: Do we want to start hardwiring the projection?

Comment (CoryMartin-NOAA) [reply src/eva/plot_tools/figure.py]: Yeah... we don't want to hard code these sort of things...

Comment (mjhossen) [reply src/eva/plot_tools/figure.py]: Is there any way to pass ccrs.PlateCarree() through self.projection.projection? I tried but failed to figure out.

Comment (CoryMartin-NOAA) [reply src/eva/plot_tools/figure.py]: See the MapProjection class...: https://github.com/JCSDA-internal/eva/blob/18f1225feba26fb40949753d5177f1c2080e18c6/src/eva/plot_tools/maps.py#L292

**Comment by danholdaway** [comment]
Can you please add an example of using this feature to the tests? I.e. add a new plot to https://github.com/JCSDA-internal/eva/blob/develop/src/eva/tests/config/testIodaObsSpaceAmsuaN19.yaml that uses the polar stereographic projection. Then upload said plot to the PR so we can see it.

**Comment by mjhossen** [comment]
Here is the plot using L4 sea-ice thickness from Copernicus. You can find more plots in https://github.com/NOAA-EMC/godas/issues/394 that compares with and without polar stereo
Screen Shot 2022-10-20 at 4 36 19 PM

**Comment by mjhossen** [comment]
> Can you please add an example of using this feature to the tests? I.e. add a new plot to https://github.com/JCSDA-internal/eva/blob/develop/src/eva/tests/config/testIodaObsSpaceAmsuaN19.yaml that uses the polar stereographic projection. Then upload said plot to the PR so we can see it.

Here is the plot using AmsuaN19 data:
Screen Shot 2022-10-26 at 4 13 15 PM

**Comment by danholdaway** [comment]
@mjhossen this all looks good. However you've caught us at a time where EMCpy is being split off from Eva. Some of the changes here therefore need to be made to EMCpy instead of Eva. Once the split is complete we'll let you know to update the PR.

**Comment by danholdaway** [comment]
@kevindougherty-noaa @CoryMartin-NOAA can you help @mjhossen with transferring these changes to EMCpy. I will close this PR since it can no longer be merged with the new structure.

**Pull request closed without merging**